nested/controllerconfig.json

```diff
@@ -48,6 +48,10 @@
         "documentation10": "    moodleDbUser                - database user for moodle",
         "documentation11": "    moodleDbPass                - database password for moodleDbUser",
         "documentation12": "    moodleAdminPass             - password for moodle admin user",
+        "documentation13": "    serviceObjective            - MS SQL porformance tier.",
+        "documentation14": "    skuTier                     - MS SQL edition tier",
```

Comment:
I've added 'premium' as an allow option, however I think the Premium edition SQL server would be over-provisioned compared to the load test results using the Standard option.

Reply:
I don't see the skuTier option change for the Premium edition...? I'm not sure if I'm seeing all the new commits you pushed after my earlier review. It's not very straightforward to do code review on GitHub...

If we are to add the Premium edition for the MS SQL DB choice, then I suppose the skuTier option for MS SQL might need to be separated, because there's no Premium SKU tier for the other two DB choices (MySQL & PostgreSQL).

Are you saying your load testing results with MS SQL were fine with the Standard edition? For how many concurrent threads, like 1600? Could you please share your load testing results for this addition separately?